validator/validate.js

```diff
@@ -0,0 +1,470 @@
+    throw new Error("The path passed must be a directory");
+  }
+}
+catch (err) {
+  process.exitCode = -1;
+  console.error("\n Invalid Path: "+err.message+"\n");
+  return;
+}
+
+/* load configuration */
+var warnings = {};
+var errors = {};
+var validSchemas = {};
+var validExamples = {};
+
+var ignoreFolders = nconf.get('fiware:ignoreFolders');
+var docFolders = nconf.get('fiware:docFolders');
+ignoreFolders = ignoreFolders.concat(['.git','node_modules']);
+var warningChecks = nconf.get('fiware:warningChecks');
+var externalSchemaFolders = nconf.get('fiware:externalSchemaFolders');
+var ignoreWarnings = (nconf.get('fiware:warnings') == 'ignore');
+var failWarnings = (nconf.get('fiware:warnings') == 'fail');
+var failErrors = !nconf.get('ajv:allErrors');
+
+//load a remote schema
+function loadSchema(uri,callback) {
+  request(uri,call);
+
+  var call = function(err, res, body) {
+    if (err || res.statusCode >= 400)
+      callback(err || new Error('Loading error: ' + res.statusCode));
+    else {
+      callback(null,JSON.parse(body));
+    }
+  }
+
+};
+
+var ajvOptions = {
+  // validation and reporting options:
+  allErrors:        nconf.get('ajv:allErrors'),
+  schemas:          {},
+  // referenced schema options:
+  missingRefs:      nconf.get('ajv:missingRefs'),
+  extendRefs:       nconf.get('ajv:extendRefs'), // recommended 'fail'
+  loadSchema:       loadSchema, // function(uri: string): Promise {}
+};
+
+
+//load additional schemas from filepath
+var addSchemas = function(fileList, method, fileType) {
+  if (!fileList) return;
+  var files = getFiles(fileList);
+  files.forEach(function (file) {
+    var schema = openFile(file, fileType);
+    method(schema);
+  });
+};
+
+//load  a list of files, supports patterns, e.g. *-schema.json
+function getFiles(args) {
+  var files = [];
+  if (Array.isArray(args)) args.forEach(_getFiles);
+  else _getFiles(args);
+  return files;
+
+  function _getFiles(fileOrPattern) {
+    if (glob.hasMagic(fileOrPattern)) {
+      var dataFiles = glob.sync(fileOrPattern, { cwd: process.cwd() });
+      files = files.concat(dataFiles);
+    } else {
+      files.push(fileOrPattern);
+    }
+  }
+};
+
+//load a JSON File
+var openFile = function (filename, suffix){
+  var json = null;
+  var file = path.resolve(process.cwd(), filename);
+  try {
+    try {
+      json = JSON.parse(fs.readFileSync(file).toString());
+    } catch(JSONerr) {
+      json = require(file);
+    }
+  } catch(err) {
+    console.error('error:  ' + err.message.replace(' module', ' ' + suffix));
+    process.exit(2);
+  }
+  return json;
+};
+
+//check if a documentation file exists in a given path
+var docExist = function (fullPath){
+  if (!fileExists(fullPath, "spec.md") && !fileExists(fullPath, "introduction.md"))
+  if (addWarning(fullPath, "does not include a documentation file name spec.md or introduction.md") && failWarnings)
+    throw new Error("Fail on Warnings: " +JSON.stringify(warnings,null, '\t'));
```

Comment:
missing space. please run a linter against your code JSHint + fixjsstyle could do the trick